README.md

```diff
@@ -82,6 +82,15 @@ User.create(email: 'xxxxxx', password: 'xxxxxx')
 5. Load test users and sample data: `bundle exec rake db:seed`
 5. Start the application: `bundle exec rails s`
 
+### Local Production Dependency Build
+The steps below will setup your local without dependencies that are not loaded in dev, staging, and production environments.
+1. `bundle install --without development test`
+2. `bundle clean --force`
+3. `bundle exec rails s`
```

Comment:
This is just running the server locally in a `development` environment, which is standard.  This doesn't test/mimic a `production` environment.

If you want this to mimic a `production` environment, you'll need the flag: 
`bundle exec rails s -e production` so that it uses the `config/environments/production.rb` configuration.

And then you could see the css/js links are broken (e.g. if you've never precompiled assets locally), so you'll need to add
`rm -f public/assets/*` to remove old/stale compiled assets.
`bundle exec rake assets:clean assets:precompile` to compile assets.

And you'll have make sure you have a `RAILS_SERVE_STATIC_FILES` environment variable (used in `config/environments/production`) set locally so that Rails will server the assets in `public/assets`.  This is _not_ set in deployed environments because `nginx` serves the static assets there.

And you'll have to make sure `database.yml` config entry for `production` isn't trying to connect to the production database, as it will fail.

Reply:
Correct, is a better title needed for this section as the intention to provide steps of how to test without development and test group dependencies?

Reply:
I'd go with something like:
> The steps below will run `GIDS` locally with only gems installed that are used in Rails `production` environments.

But I'm still having trouble understanding what this is trying to accomplish.
What are you testing for with these commands that would be different than running the site locally after plain old `bundle install`?

Reply:
Removed the section